lib/prx_auth/rails/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module PrxAuth
   module Rails
-    VERSION = "5.0.1"
+    VERSION = "5.1.0"
```

Comment:
Technically, a breaking change?  But, we know which apps use this gem, so maybe a minor version is good enough.

Reply:
agree!